src/schoology-extractor/data_extractor/lib/assignments.py

```diff
@@ -0,0 +1,28 @@
+from .request_client import RequestClient
+
+
+class Assignments:
+    """
+    The Assignments class contains methods to get assignments' information from Schoology API
+
+    Args:
+        SCHOOLOGY_KEY (str): The consumer key given by Schoology
+        SCHOOLOGY_SECRET (str): The consumer secret given by Schoology
+
+    Attributes:
+        request_client (RequestClient): The client used for authenticate and send requests to Schoology
+
+    """
+
+    def __init__(self, SCHOOLOGY_KEY, SCHOOLOGY_SECRET):
+        self.request_client = RequestClient(SCHOOLOGY_KEY, SCHOOLOGY_SECRET)
+
+    def get_by_section_id_array(self, section_id_array):
```

Comment:
please add guard assertion to ensure that `section_id_array` is not `None`. Might also make sense to ensure that it is a list. Finally, might consider removing the word "array" from the argument name. Maybe just `section_ids`?

Reply:
I will make the change